bip-musig2/sign_verify_vectors.json

```diff
@@ -0,0 +1,142 @@
+		"020000000000000000000000000000000000000000000000000000000000000009"
+	],
+	"aggnonces": [
+		"028465FCF0BBDBCF443AABCCE533D42B4B5A10966AC09A49655E8C42DAAB8FCD61037496A3CC86926D452CAFCFD55D25972CA1675D549310DE296BFF42F72EEEA8C9",
+		"000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+		"048465FCF0BBDBCF443AABCCE533D42B4B5A10966AC09A49655E8C42DAAB8FCD61037496A3CC86926D452CAFCFD55D25972CA1675D549310DE296BFF42F72EEEA8C9",
+		"028465FCF0BBDBCF443AABCCE533D42B4B5A10966AC09A49655E8C42DAAB8FCD61020000000000000000000000000000000000000000000000000000000000000009",
+		"028465FCF0BBDBCF443AABCCE533D42B4B5A10966AC09A49655E8C42DAAB8FCD6102FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFEFFFFFC30"
+	],
+	"msgs": [
+		"F95466D086770E689964664219266FE5ED215C92AE20BAB5C9D79ADDDDF3C0CF",
+		""
+	],
+	"valid_test_cases": [
+		{
+			"key_indices": [0, 1, 2],
+			"nonce_indices": [0, 1, 2],
+			"aggnonce_index": 0,
+			"msg_index": 0,
+			"signer_index": 0,
+			"expected": "68537CC5234E505BD14061F8DA9E90C220A181855FD8BDB7F127BB12403B4D3B"
+		},
+		{
+			"key_indices": [1, 0, 2],
+			"nonce_indices": [1, 0, 2],
+			"aggnonce_index": 0,
+			"msg_index": 0,
+			"signer_index": 1,
+			"expected": "2DF67BFFF18E3DE797E13C6475C963048138DAEC5CB20A357CECA7C8424295EA"
+		},
+		{
+			"key_indices": [1, 2, 0],
+			"nonce_indices": [1, 2, 0],
+			"aggnonce_index": 0,
+			"msg_index": 0,
+			"signer_index": 2,
+			"expected": "0D5B651E6DE34A29A12DE7A8B4183B4AE6A7F7FBE15CDCAFA4A3D1BCAABC7517"
+		},
+		{
+			"key_indices": [0, 1],
+			"nonce_indices": [0, 3],
+			"aggnonce_index": 1,
+			"msg_index": 0,
+			"signer_index": 0,
+			"expected": "8D5E0407FB4756EEBCD86264C32D792EE36EEB69E952BBB30B8E41BEBC4D22FA",
+			"comment": "Both halves of aggregate nonce correspond to point at infinity"
+		},
+		{
+			"key_indices": [0, 1, 2],
+			"nonce_indices": [0, 1, 2],
+			"aggnonce_index": 0,
+			"msg_index": 1,
+			"signer_index": 0,
+			"expected": "0167931A502D28A128DDAC1DC2A53296F00DA663CCDDC9673D79020091A3A58A",
+			"comment": "Empty message"
+		}
+	],
+	"sign_error_test_cases": [
+		{
+			"key_indices": [1, 0, 3],
+			"aggnonce_index": 0,
+			"msg_index": 0,
+			"comment": "Signer 2 provided an invalid public key"
+		},
+		{
+			"key_indices": [1, 2, 0],
+			"aggnonce_index": 2,
+			"msg_index": 0,
+			"comment": "Aggregate nonce is invalid due wrong tag, 0x04, in the first half"
+		},
+		{
+			"key_indices": [1, 2, 0],
+			"aggnonce_index": 3,
+			"msg_index": 0,
+			"comment": "Aggregate nonce is invalid because the second half does not correspond to an X coordinate"
+		},
+		{
+			"key_indices": [1, 2, 0],
+			"aggnonce_index": 4,
+			"msg_index": 0,
+			"comment": "Aggregate nonce is invalid because the second half does not correspond to an X coordinate"
```

Comment:
comment is wrong. It should be "Aggregate nonce is invalid because second half exceeds field size".